sd/test/teamcity.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/python
+
+import sys
+import os
+import yaml
+import shutil
+import argparse
+import subprocess
+from traceback import print_exc
+
+#Log class, use it, not print
+class Log:
```

Comment:
is this the log class that is expected to be used everywhere in the project? We already have a logger here: https://github.com/ablab/stringdecomposer/blob/devel/sd/standard_logger.py
Can you please merge them?

Reply:
Example of usage: https://github.com/ablab/stringdecomposer/blob/api/sd/cluster_sequences.py